CI Failure: Admittance controller build error

Our rolling CI currently fails with the following error:

```console
 In file included from /root/upstream_ws/src/ros2_controllers/admittance_controller/src/admittance_controller.cpp:17:
  /root/upstream_ws/src/ros2_controllers/admittance_controller/include/admittance_controller/admittance_controller.hpp:100:37: error: ‘controller_interface::return_type admittance_controller::AdmittanceController::update_reference_from_subscribers(const rclcpp::Time&, const rclcpp::Duration&)’ marked ‘override’, but does not override
    100 |   controller_interface::return_type update_reference_from_subscribers(
        |                                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  In file included from /opt/ros/rolling/include/class_loader/class_loader/class_loader_core.hpp:58,
                   from /opt/ros/rolling/include/class_loader/class_loader/class_loader.hpp:55,
                   from /opt/ros/rolling/include/class_loader/class_loader/multi_library_class_loader.hpp:52,
                   from /opt/ros/rolling/include/pluginlib/pluginlib/class_loader.hpp:38,
                   from /root/upstream_ws/src/ros2_controllers/admittance_controller/include/admittance_controller/admittance_rule.hpp:32,
                   from /root/upstream_ws/src/ros2_controllers/admittance_controller/include/admittance_controller/admittance_controller.hpp:28,
                   from /root/upstream_ws/src/ros2_controllers/admittance_controller/src/admittance_controller.cpp:17:
  /opt/ros/rolling/include/class_loader/class_loader/meta_object.hpp: In instantiation of ‘B* class_loader::impl::MetaObject<C, B>::create() const [with C = admittance_controller::AdmittanceController; B = controller_interface::ChainableControllerInterface]’:
  /opt/ros/rolling/include/class_loader/class_loader/meta_object.hpp:216:7:   required from here
  /opt/ros/rolling/include/class_loader/class_loader/meta_object.hpp:218:12: error: invalid new-expression of abstract class type ‘admittance_controller::AdmittanceController’
    218 |     return new C;
```

This is caused by this ros2_control API change: https://github.com/ros-controls/ros2_control/pull/846. It was already fixed in ros2_controllers (https://github.com/ros-controls/ros2_controllers/pull/553), which is why we only fail on main and not testing.

--- a/Universal_Robots_ROS2_Driver-not-released.rolling.repos
+++ b/Universal_Robots_ROS2_Driver-not-released.rolling.repos
@@ -3,12 +3,4 @@
 # requires a newer version than the one currently released to the target distributions.
 # Once Upstream packages are released and synced to the target distributions in the required
 # version, the entry in this file shall be removed again.
-repositories:
-  moveit2:
-    type: git
-    url: https://github.com/ros-planning/moveit2.git
-    version: main
-  ros2_controllers:
-    type: git
-    url: https://github.com/ros-controls/ros2_controllers
-    version: master
+repositories: []

